### PR TITLE
Use test binary for testing of KubeVirt master

### PR DESCRIPTION
**Pull request by dhiller**
Instead of building the test binary from the master branch, we consume
the binary that is built beside the KubeVirt nightly build.

Therefore we do not require the kubevirt builder image anymore and
switch to using a basic golang image instead.

**Note: Requires https://github.com/kubevirt/project-infra/pull/552 to work properly!**

### CODE REVIEW

**Comment by dhiller** [comment]
At least the running of the tests with the binary works now: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/12371/rehearse-12371-periodic-ci-kubevirt-kubevirt-master-e2e-nested-virt/1314548104074104832

File: ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml [hunk 1]
```diff
@@ -29,49 +21,67 @@ tag_specification:
...
+    - as: nightly-build-azure4
+      commands: |
+        set -euo pipefail
 
-    release_base_url="https://gcsweb.apps.ovirt.org/gcs/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt"
-    release_date=$(curl -L "${release_base_url}/latest")
-    release_url="${release_base_url}/${release_date}"
-    commit=$(curl -L "${release_url}/commit")
+        echo "Downloading oc binary"
+        # FIXME: transfer into Dockerfile for ci
```
Comment (ashleyschuett) [inline ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml:34]: is this a later to do?

Comment (dhiller) [reply ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml:34]: Actually I'm not really sure what to do with this. The DPTP team told me they are working on this to not leave the tedious download burden on the user, so this is more of a reminder for me to nag there sometime later ;-)

**Comment by dhiller** [comment]
/hold cancel 

kubevirt/project-infra/pull/552 just got merged, so we should be good to go.

**Pull request merged**